.github/workflows/code_coverage.yml

```diff
@@ -0,0 +1,29 @@
+name: Code coverage
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  code-coverage:
+    name: Code coverage
+    runs-on: ubuntu-latest
+
+    container:
+      image: xd009642/tarpaulin:develop-nightly
+      options: --security-opt seccomp=unconfined
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Generate code coverage
```

Comment:
Maybe we need to add cache to speed up compilation, similar to what we have here?
[source](https://github.com/astroport-fi/astroport-governance/blob/a41ee892b2e8aaa8cd3722cbb4e4a9c7fdb87957/.github/workflows/tests_and_checks.yml#L25)

Reply:
Done, thanks